### PR TITLE
Change installation path of cfn-bootstrap

### PR DESCRIPTION
**Pull request by francesco-giordano**
### Description of changes
* Change the installation path to be in a virtualenv.
* Creation of the profile file pcluster.sh to automatically include the cfn-bootstrap binary in the PATH

### Tests
* Automated image build and integration tests.

Please review the [guidelines for contributing](../CONTRIBUTING.md) and [Pull Request Instructions](https://github.com/aws/aws-parallelcluster/wiki/Git-Pull-Request-Instructions).

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: attributes/default.rb [hunk 1]
```diff
@@ -47,12 +47,16 @@
...
 default['cluster']['awsbatch_virtualenv'] = 'awsbatch_virtualenv'
+# Virtualenv cfn-bootstrap Name
+default['cluster']['cfn_bootstrap_virtualenv'] = 'cfn_bootstrap_virtualenv'
 # Cookbook Virtualenv Path
 default['cluster']['cookbook_virtualenv_path'] = "#{node['cluster']['system_pyenv_root']}/versions/#{node['cluster']['python-version']}/envs/#{node['cluster']['cookbook_virtualenv']}"
 # Node Virtualenv Path
 default['cluster']['node_virtualenv_path'] = "#{node['cluster']['system_pyenv_root']}/versions/#{node['cluster']['python-version']}/envs/#{node['cluster']['node_virtualenv']}"
 # AWSBatch Virtualenv Path
 default['cluster']['awsbatch_virtualenv_path'] = "#{node['cluster']['system_pyenv_root']}/versions/#{node['cluster']['python-version']}/envs/#{node['cluster']['awsbatch_virtualenv']}"
+# AWSBatch Virtualenv Path
```
Comment (lukeseawalker) [inline attributes/default.rb]: wrong comment

File: cookbooks/aws-parallelcluster-install/recipes/python.rb [hunk 2]
```diff
@@ -49,7 +56,15 @@
...
+  creates "#{node['cluster']['cfn_bootstrap_virtualenv_path']}/bin/cfn-hup"
 end
+
+# Add cfn_bootstrap virtualenv to default path
+template "/etc/profile.d/pcluster.sh" do
+  source "base/pcluster.sh.erb"
+  owner 'root'
+  group 'root'
+  mode '0644'
+end
```
Comment (lukeseawalker) [inline cookbooks/aws-parallelcluster-install/recipes/python.rb]: leave an empty line at the end

Comment (francesco-giordano) [reply cookbooks/aws-parallelcluster-install/recipes/python.rb]: Done

File: attributes/default.rb (hunk 1 above)
Comment (francesco-giordano) [reply attributes/default.rb]: Done

**Review by lukeseawalker — APPROVED** [review]

**Pull request merged**